README.md

```diff
@@ -1,4 +1,4 @@
-# diaspora*
+# diaspora\*
```

Comment:
I'm not sure if escaping this one is needed, but it's better be consistent, right?